### PR TITLE
Update AD-FS-FAQ.md

### PR DESCRIPTION
**Pull request by gaubert-ms**
There is a mistake. It's not supported to place third party proxy in front of WAP.

### CODE REVIEW

**Comment by eross-msft** [comment]
@billmath  Please review and let me know if it's okay to publish. Thanks!!

File: WindowsServerDocs/identity/ad-fs/overview/AD-FS-FAQ.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ AD FS provides an extensible mechanism for 3rd party MFA providers to integrate.
 The list of vendors that have notified Microsoft are published at [MFA providers for AD FS](../operations/Configure-Additional-Authentication-Methods-for-AD-FS.md).  There may always be providers available that we do not know about and we will update the list as we learn about them.
 
 ### Are third party proxies supported with AD FS?
-Yes, third party proxies can be placed in front of the Web Application Proxy, but any third party proxy must support the [MS-ADFSPIP protocol](https://msdn.microsoft.com/library/dn392811.aspx) to be used in place of the Web Application Proxy.
+Yes, third party proxies can be placed in front of AD FS, but any third party proxy must support the [MS-ADFSPIP protocol](https://msdn.microsoft.com/library/dn392811.aspx) to be used in place of the Web Application Proxy.
```
Comment (illfated) [inline WindowsServerDocs/identity/ad-fs/overview/AD-FS-FAQ.md:83]: This MSDN link is permanently redirected to docs.microsoft.com with this URL:
```suggestion
Yes, third party proxies can be placed in front of AD FS, but any third party proxy must support the [MS-ADFSPIP protocol](https://docs.microsoft.com/openspecs/windows_protocols/ms-adfspip/76deccb1-1429-4c80-8349-d38e61da5cbb) to be used in place of the Web Application Proxy.
```
And yes, the docs.microsoft.com page supports language agnostic localization.

**Comment by illfated** [comment]
BTW, the change makes good sense to me, but I don't know the topic well enough to confirm. Waiting for Bill Mathers to confirm that the change is correct.

**Comment by eross-msft** [comment]
@gaubert-ms I can work with you to merge this update, but as a Microsoft employee, please use the private repo, https://github.com/MicrosoftDocs/windowsserverdocs-pr in the future. You will have more options from the writing team by going through there.

@billmath  Please review this text and let me know if I can merge it. Thanks!

**Comment by billmath** [comment]
Looks good.  

#sign-off

**Comment by gaubert-ms** [comment]
Hi Elizabeth,

Thank you for your answer. I tried the link you send me but I get a 404. Is it correct?

Regards,

Guillaume.

**Comment by illfated** [comment]
@gaubert-ms : You probably need to access the internal Microsoft internal PR repository (https://github.com/MicrosoftDocs/windowsserverdocs-pr) directly from this repository by visiting pages where this link is posted. I don't know exactly what happens to link translation in the e-mails you read from GitHub, so maybe the links don't open correctly if you click a link in one of your e-mail messages.

**Comment by eross-msft** [comment]
@gaubert-ms  Yes, the 404 error is normal, I see it too :) It just means that you have to sign in to your GitHub account before you can access the repo. The repo needs to make sure you have the right credentials...after that, you can use the instructions here: https://github.com/MicrosoftDocs/windowsserverdocs-pr/blob/master/Contributor-guide/github-browser-updates.md (if you want to edit, but don't want to use a code editor) or here: https://github.com/MicrosoftDocs/windowsserverdocs-pr/blob/master/Contributor-guide/edit-existing-using-github.md (to edit using Visual Studio Code). Hope this helps! Thank you for all of your effort so far, it's greatly appreciated!

**Pull request merged**